src/main/java/com/meta/chatbridge/message/Message.java

```diff
@@ -29,4 +31,11 @@ enum Role {
     USER,
     SYSTEM
   }
+
+  default Identifier conversationId() {
```

Comment:
no they're different, conversation id is an identifier formed from the sender and receiver ids, so it will be the same for all messages between those two parties. Instance id is and id for the message in particular. These will be more important when we start storing the messages in a DB